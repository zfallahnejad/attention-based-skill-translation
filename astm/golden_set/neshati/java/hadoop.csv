1279787,hadoop
540873,hadoop
614157,hadoop
1206837,hadoop
1332690,hadoop
2516301,hadoop
352268,hadoop
610305,hadoop
1038826,hadoop
1708119,hadoop
3276920,hadoop
213901,hadoop
1050422,hadoop
