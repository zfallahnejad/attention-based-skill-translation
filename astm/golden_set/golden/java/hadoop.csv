1279787,82,45,0.6456692913385826
540873,50,38,0.5681818181818182
614157,41,56,0.422680412371134
1206837,29,40,0.42028985507246375
1332690,29,13,0.6904761904761905
2516301,15,8,0.6521739130434783
352268,14,13,0.5185185185185185
610305,13,13,0.5
1038826,12,3,0.8
1708119,12,15,0.4444444444444444
3276920,12,13,0.48
204665,11,13,0.4583333333333333
213901,11,6,0.6470588235294118
1050422,11,10,0.5238095238095238
2438110,10,8,0.5555555555555556
2531314,10,9,0.5263157894736842
